helpers/get-themes.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+module.exports = function (plugins, config) {
+  return function (name) {
+    const themeName = name || false;
+    const themes = Object.keys(config.themes);
+
+    if (themes.length === 0) {
+      throw new plugins.util.PluginError({
+        plugin : 'config',
+        message: plugins.errorMessage('You have to create <magento root>/dev/tools/frontools/configs/themes.json or run "gulp setup"')
+      });
+    }
+
+    if (themeName && themes.indexOf(themeName) === -1) {
+      throw new plugins.util.PluginError({
+        plugin : 'config',
+        message: plugins.errorMessage('Theme provided using --theme <theme-name> is not available in <magento root>/dev/tools/frontools/configs/themes.json')
+      });
+    }
+
+    return themeName ? themes[themeName] : themes;
+  }
```

Comment:
I'd simplify this and just add `required` param to `config-loader`.
Passing theme name to function named `getThemes` is weird, b/c it should return themes object, not a single theme or error about this theme.